tests/platform_helper/utils/test_validation.py

```diff
@@ -10,16 +10,20 @@
 from moto import mock_aws
 from schema import SchemaError
 
+from dbt_platform_helper.constants import PLATFORM_CONFIG_FILE
```

Comment:
Should we rely on this definition in the tests?

You could change it and I think the tests would still pass.

I'm wondering if the tests should have their own declaration of this?

Reply:
My thinking is that these tests don't care what the filename is just the logic of what happens when that is validated. If we decide to change the filename we don't then have to work out if these tests broke because of logic, or broke due to just the filename change (which we don't care about here)

Reply:
I mean, perhaps a separate test "test_filename_is_pipeline_config_yml" that hardcodes the name would give a single test that would challenge people to decide if this is really what they wanted to do (in case of introducing a typo or something by accident)